lib/ws_servers/api/handlers/strategy/strategy_manager.js

```diff
@@ -172,27 +172,36 @@ class StrategyManager {
    * @param {boolean} [strategyArgs.includeTrades] - option to include trades or not
    * @param {number} [strategyArgs.seedCandleCount] - number of candles to seed before strategy execution
    */
-  async execute (strategy = {}, strategyOpts = {}) {
+  async execute (parsedStrategy = {}, strategyOpts = {}) {
```

Comment:
it would be nice to update the jsdoc above

Reply:
done